test/loading.jl

```diff
@@ -692,7 +692,7 @@ mktempdir() do dir
     mkpath(vpath)
     script = "@assert startswith(Base.active_project(), $(repr(vpath)))"
     cmd = `$(Base.julia_cmd()) --startup-file=no -e $(script)`
-    cmd = addenv(cmd, "JULIA_DEPOT_PATH" => dir)
+    cmd = addenv(cmd, "JULIA_DEPOT_PATH" => dir, "JULIA_LOAD_PATH" => ":")
```

Comment:
This needs to be `";"` for windows?
```
Error in testset loading:
Test Failed at C:\buildkite-agent\builds\win2k22-amdci6-7\julialang\julia-master\julia-ff1a431e64\share\julia\test\loading.jl:697
  Expression: success(cmd)
```

Reply:
Think so: https://github.com/JuliaLang/julia/blob/f8dd16e5c1b2f372bda0f787a4fefe4911d7f787/test/loading.jl#L701